### DISCUSSION TITLE
How to integrate Github Copilot?

### DISCUSSION THREAD

**Discussion by Nebucatnetzer** [karthink/gptel#371] [Q&A]
What would be the way to integrate Github Copilot or is this already possible?
I’ve searched for but didn’t find any mentions of it.
There is another plugin that integrates with it but I don’t like the way how it behaves and would like to test this one.

**Comment by Nebucatnetzer** [comment]
It looks like Copilot is only available through a language server.

**Comment by karthink** [comment]
> It looks like Copilot is only available through a language server.

Yes, gptel has no "intelligence", it just plumbs text from buffers to an LLM and back in the simplest way possible.
